### ISSUE TITLE
[🐛 BUG] - Unable to open files

### ISSUE THREAD

**Issue by Unknow-max** [alexkim205/G-Desktop-Suite#75]
**Describe the bug**
Currently when I try to open a file from the program, the drive app open obviously the preview, like this:
![immagine](https://user-images.githubusercontent.com/59534318/82752964-264cb900-9dc2-11ea-9c1d-cc22dcc8cd0a.png)

But when I try to open the file clicking "Open with sheets" it don't let me open the file:
![immagine](https://user-images.githubusercontent.com/59534318/82752973-48ded200-9dc2-11ea-8c58-cf46c7ab617c.png)
It simply do nothing at all.
A

**To Reproduce**
1. Try to open a preview of a file
2. Click on "open with..."
3. As you can see, the file will not open

**G Desktop Suite:**
 - Version [e.g. 0.2.1]

**Platform:**
 - OS: Windows
 - Version : 10
Version of the app: G Desktop Suite - 0.3.1

**Comment by mding5692** [comment]
Think this is associated with unable to open links within child windows?

**Comment by alexkim205** [comment]
Yeah I think you're right. @Unknow-max what is the expected behavior for this issue? For example, when you try this in your normal browser, does google drive try to open a new tab with the newly created doc? Or does it return you back to the main drive page?

**Comment by Unknow-max** [comment]
**Ok, first of all I did some research on the issue, and I found some interesting facts.**

1. If you try to open from the preview with open external link in browsers checked, like so:
![immagine](https://user-images.githubusercontent.com/59534318/82806947-1fd44500-9e87-11ea-9549-b0e15e806e3d.png)
When you will try to click the button "open with..." Windows system will say:
![immagine](https://user-images.githubusercontent.com/59534318/82807079-6164f000-9e87-11ea-9d6c-842983c62a0b.png)
(translated from italian language) "You will need  a new app to open this link", and it will not letting you to move on with "open link with this app".

2. If you try to open, instead, with "open-external link in browsers" when you click "open the document" it will simply do nothing at all.

Answering your question, usually the normal behavior of Google Drive is to open the document in another tab when you press from the preview "open with.....(name of the program).

**Comment by Unknow-max** [comment]
In general, in the app all work fine (the created document will open in new tab ecc...) except when you try to open a document already created.

**Comment by Unknow-max** [comment]
**Noticed addiotionally one thing!!!!**

![immagine](https://user-images.githubusercontent.com/59534318/82807628-71c99a80-9e88-11ea-8ca0-0872523ae1a9.png)
See there, if you try to right click on a document> open with.....> "some program" on the bottom on the screen will appear a popup that will say (translated from italian langauge): The browser ha

**Comment by Unknow-max** [comment]
@alexkim205 Are there some updates about the issue?
